index.html

```diff
@@ -228,5 +228,37 @@ <h3>Languages</h3>
             </div>
         </div>
     </div>
-</body>
-</html>
\ No newline at end of file
+    <form class="former-block">
+                <div class="form-block">
+                    <section>
+                        <h1 class="contact">Contact me</h1>
+                            <p class="paragraph">If you have an application y ou are interested in developing,
+                                a feature that you need built or   a project that needs 
+                                coding. I’d love to help with it 
+                            </p>
```

Comment:
- [ ] Kindly add `font-family` to these texts to match the design in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200880599-ea2e6888-8b06-4420-8afc-44e78d3e2932.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200880635-ca183c56-c4fb-4aba-8a56-27625efca7db.png)